department-of-veterans-affairs/va.gov-cms
# VACMS-4763: CER-D9-Patch-Updated

…ror that occurs when creating new CERs

## Description

#4763

## Testing done


## Screenshots


## QA steps

- Login to preview environment
- Navigate here: `/admin/config/content/cer/add`
- [x] Confirm no errors appear and you can create and save a new CER with no issues. 

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
